english/coffee/cookie/bake-cookies.md

```diff
@@ -5,3 +5,9 @@ You satiate yourself with a pan
 full of freshly baked cookies.
 
 Yum.
+
+After having coffee and cookies, you would want to:
+
+[Go for a walk](walk\walk.md)
```

Comment:
You should use a forward slash (`/`), not a backward slash (`\`) in these urls. They are also case sensitive, so make sure you use capital letters if your directories and files have capitals in their names.